tests/Feature/GenerateTest.php

```diff
@@ -38,29 +39,45 @@ public function test_spec_is_json()
         $this->assertEquals('My JSON:API', $spec['info']['title']);
     }
 
-    public function test_spec_file_generated()
+    /**
+     * @dataProvider disksDataProvider
+     */
+    public function test_spec_file_generated(string $disk)
     {
+        config(['openapi.filesystem_disk' => $disk]);
```

Comment:
For as far as I'm concerned we can assume all storage drivers just work - we only create a file, nothing fancy - so we don't need to test different storage drivers. That means the data provider can be removed.